docs/src/snap/howto/networking/alternative-cni.md

```diff
@@ -0,0 +1,107 @@
+# How to use an alternative CNI
+
+While {{product}} ships with a default [Container Network Interface] (CNI) that
+we guarantee is compatible with our distribution, you may want to use a
+different CNI plugin for your specific networking requirements. This guide
```

Comment:
Can you include a link to an upstream page that lists supported alternative CNIs?